.env.example

```diff
@@ -0,0 +1,5 @@
+DB_HOST=localhost
+DB_USER=root
+DATABASE=mysite
+DB_PASSWORD=732001
+DB_PORT=3306
```

Comment:
Mình dùng editor nào vậy e? 
Nếu pycharm thì mình config để format auto thêm indentation (thêm line trống cuối mỗi file) tránh bị lỗi indent cảnh báo của github như này e nhé
https://www.jetbrains.com/help/pycharm/code-style-sql-tabs-and-indents.html